app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt

```diff
@@ -1,45 +1,111 @@
 import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.lerp
 import androidx.compose.ui.unit.times
 import androidx.compose.ui.zIndex
 import androidx.hilt.navigation.compose.hiltViewModel
+import com.hammer.talkbbokki.R
+import com.hammer.talkbbokki.presentation.main.CategoryLevel
+import com.hammer.talkbbokki.ui.theme.TalkbbokkiTypography
 import kotlin.math.absoluteValue
 import kotlin.math.roundToInt
 
+val level = "Level1"
+
 @Composable
 fun TopicListRoute(
     modifier: Modifier = Modifier,
-    onClickToDetail: (id : String) -> Unit,
+    onClickToDetail: (id: String) -> Unit,
     viewModel: TopicListViewModel = hiltViewModel()
 ) {
+    TopicListScreen(onClickToDetail = onClickToDetail, viewModel = viewModel)
+}
+
+@Composable
+fun TopicListScreen(onClickToDetail: (id: String) -> Unit, viewModel: TopicListViewModel) {
+    var selectedIdx by remember { mutableStateOf("0") }
     val topicList by viewModel.topicList.collectAsState()
-    TopicListScreen(onClickToDetail = onClickToDetail)
+
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(CategoryLevel.valueOf(level).backgroundColor)
+    ) {
+        TopicListHeader()
+        TopicList(onFocusedCardChange = { idx -> selectedIdx = idx })
+        SelectBtn(
+//            viewCnt = viewModel.todayViewCnt,
+            viewCnt = 3,
+            onCardClicked = {
+                viewModel.setTodayViewCnt()
+                onClickToDetail(selectedIdx)
+            },
+            Modifier.align(Alignment.BottomCenter)
+        )
+    }
+}
+
+@Composable
+fun TopicListHeader() {
+    Column(
+        modifier = Modifier
+            .padding(
+                vertical = dimensionResource(id = R.dimen.vertical_padding),
+                horizontal = dimensionResource(id = R.dimen.horizontal_padding)
+            )
+            .fillMaxWidth()
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.ic_close),
+            contentDescription = null,
+            modifier = Modifier
+                .align(Alignment.End)
+                .size(24.dp, 24.dp),
+        )
+        Spacer(modifier = Modifier.height(24.dp))
+        stringResource(id = CategoryLevel.valueOf(level).title).split("\n")
+            .forEachIndexed { index, s ->
+                Text(
+                    text = s,
+                    style = TalkbbokkiTypography.h2_bold,
+                    color = if (index == 0) Color.Black else Color(0x80000000)
+                )
+            }
+    }
 }
 
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
-fun TopicListScreen(modifier: Modifier = Modifier, onClickToDetail: (id : String) -> Unit) {
+fun TopicList(onFocusedCardChange: (idx: String) -> Unit) {
+
```

Comment:
🚫 **[ktlint]** <no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty